### PR TITLE
Change test robots to use points

### PR DESCRIPTION
**Pull request by liquidmetal9015**
## Description
This should be considered after #1393, as it is branched off of that branch and is a minor change after that. I'm not exactly sure how to handle dependent/sequential pull requests as it seems there is really no good solution for it aside from just adding changes to the previous pull request. I guess I'm just doing this for now.

Currently the testRobot object the gameplay tests system uses x and y as values, where it is seeming like using robocup.Point to be more ergonomic for quickly defining points and offsets

## Steps to test
Basically just make sure that the gameplay test system works as intended.

### CODE REVIEW

**Comment by liquidmetal9015** [comment]
I'm going to just close these and make one big one when it's ready.
